package.json

```diff
@@ -39,7 +38,7 @@
   "devDependencies": {
     "@balena/lint": "^4.1.1",
```

Comment:
Can you upgrade this to 5.x as part of the major, since it drops support for nodejs 8